pubspec.yaml

```diff
@@ -35,6 +35,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.8
+  mockito: ^5.0.0
```

Comment:
let us use https://pub.dev/packages/mocktail instead. Please update the tests with Mocktail

Reply:
Mockito was originally what we were gonna use but we did not end up using it for our test cases. Do you still want me to add tests that use Mocktail. I also added Mocktail to dependencies.

Reply:
Yes update to mocktail